application/client/src/app/ui/views/toolbar/chart/chart/component.ts

```diff
@@ -33,7 +33,7 @@ export class ViewChartCanvas extends ChangesDetector implements AfterViewInit, O
 
     public ngAfterViewInit() {
         this.state.bind(this, this.session, this.vcRef.element.nativeElement as HTMLElement);
-        this.state.init();
+        this.state.init(this.ilc());
```

Comment:
with `bind` you already sent `this`, which includes `ilc()` as well. No needs to send `this.ilc()` again with `init`